scripts/generate/generate-src.ts

```diff
@@ -4,10 +4,10 @@ import path from 'path';
 import inquirer from 'inquirer';
 
 export enum Template {
-  Vanilla = 'vanilla',
-  React = 'react',
-  VanillaTypescript = 'vanilla-typescript',
-  ReactTypescript = 'react-typescript',
+  Vanilla = 'js',
+  React = 'js+react',
+  VanillaTypescript = 'ts',
+  ReactTypescript = 'ts+react',
```

Comment:
I'm not super sold on the `+` separator but I have absolutely no valid reasoning for it. However I do like adding the more structured approach with language then framework 👍